config/default/field.field.node.hs_basic_page.field_hs_page_components.yml

```diff
@@ -31,78 +30,74 @@ settings:
   handler_settings:
     target_bundles:
       hs_gradient_hero: hs_gradient_hero
+      hs_priv_collection: hs_priv_collection
       hs_priv_text_area: hs_priv_text_area
+      hs_row: hs_row
       hs_spotlight: hs_spotlight
       hs_timeline_item: hs_timeline_item
-      hs_webform: hs_webform
-      hs_priv_collection: hs_priv_collection
-      hs_row: hs_row
     negate: 1
     target_bundles_drag_drop:
       hs_accordion:
         weight: -44
         enabled: false
       hs_banner:
-        weight: -43
+        weight: -37
         enabled: false
       hs_callout_box:
-        weight: -39
+        weight: -43
         enabled: false
       hs_carousel:
-        weight: -40
+        weight: -38
         enabled: false
       hs_clr_bnd:
-        weight: -37
+        weight: -42
         enabled: false
       hs_collection:
-        weight: -38
+        weight: -41
         enabled: false
       hs_gradient_hero:
-        weight: -30
+        weight: -40
         enabled: true
       hs_gradient_hero_slider:
-        weight: -41
+        weight: -39
         enabled: false
       hs_hero_image:
-        weight: -42
+        weight: -36
         enabled: false
       hs_postcard:
-        weight: -35
+        weight: -34
         enabled: false
       hs_priv_collection:
-        weight: -25
+        weight: -33
         enabled: true
       hs_priv_text_area:
-        weight: -29
+        weight: -32
         enabled: true
       hs_row:
-        weight: -24
+        weight: -31
         enabled: true
       hs_spotlight:
-        weight: -28
+        weight: -30
         enabled: true
       hs_sptlght_slder:
-        weight: -34
+        weight: -29
         enabled: false
       hs_testimonial:
-        weight: -33
+        weight: -28
         enabled: false
       hs_text_area:
         weight: -45
         enabled: false
       hs_timeline:
-        weight: -32
+        weight: -26
         enabled: false
       hs_timeline_item:
         weight: -27
         enabled: true
       hs_view:
-        weight: -31
+        weight: -25
         enabled: false
-      hs_webform:
```

Comment:
In overridden config, try to include only the lines necessary for the the `hs_webform` paragraph removal. All other changes might be different from site to site and it's better to leave them out of the PR.

Reply:
Removed other changes.